config/config.go

```diff
@@ -34,6 +34,7 @@ type ViperConfig struct {
 		Keywords    []string
 		Path        string
 		Tags        []string
+		Enabled     *bool
```

Comment:
I think the best name is either `enabled` or `disabled`. I'm not sure which the most intuitive.